src/client/common/installer.ts

```diff
@@ -146,9 +143,9 @@ ProductTypes.set(Product.yapf, ProductType.Formatter);
 ProductTypes.set(Product.rope, ProductType.RefactoringLibrary);
 
 export class Installer implements vscode.Disposable {
-    private static terminal: vscode.Terminal;
+    private static terminal: vscode.Terminal | undefined | null;
```

Comment:
Out of curiosity, what does this pattern do?

Reply:
Allow it to be nullable (i'm slowly introducing some strict checks.
I don't want to turn this on globally, else we'll have 100s of errors.
[New --strict master option](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-3.html)

Reply:
Is the errors from dependencies or just old code that needs an update?